jax_galsim/gaussian.py

```diff
@@ -57,10 +57,6 @@ def __init__(self, half_light_radius=None, sigma=None, fwhm=None, flux=1.0, gspa
         else:
             super().__init__(sigma=sigma, flux=flux, gsparams=gsparams)
 
-        self._sigsq = self.sigma**2
```

Comment:
same question, is the explanation that sigma will be stored as a variable of the parent GSObject?